### ISSUE TITLE
Finalize Conference Theme

### ISSUE THREAD

**Issue by dcwalk** [ournetworks/2018#22]
Here is an issue to collect ideas for a theme.

2017: "the past, present, and future of building our own network infrastructures"
---
2018 brainstorming (concepts that I think would be interesting to explore):

- being/becoming network **neighbours**
- **making** network infrastructures
- **rebuilding** and/or **repairing** existing infrastructure
- **decentralization is not enough** (I think about that MIT report "Defending Internet Freedom through Decentralization: Back to the Future?" and all the issues it highlighted quite a bit)
- lets **do it with others ([DIWO](http://archive.furtherfield.org/projects/diwo-do-it-others-resource))** (as opposed to DIY?)
- Something about **geography**, **place**, or **situation** as a starting point

**Comment by dcwalk** [comment]
As we mentioned wanting to finalize this weekend, I'm wondering @benhylau @garrying @patcon @jkms if you have any thoughts?

I'll do some writing around a theme tomorrow morning

**Comment by benhylau** [comment]
I would like the conference to explore what it means to take a _DIWO approach_, and a focus on sensible next steps. For example, from the report [Defending Internet Freedom through Decentralization: Back to the Future?](http://dci.mit.edu/assets/papers/decentralized_web.pdf)

>Reduced market centralization combined with increased transparency, rather than full "decentralization" of publishing infrastructure, might be what is needed to rebalance the current equation.

Discuss whether this is an appropriate next step and what do the actionables look like? How we can involve people of different skillsets?

**Comment by dcwalk** [comment]
Something about DIWO:
- https://medium.com/cultura-del-hacer/se-puede-ser-un-hacker-en-la-escuela-ded67ddb7f40
> Pasamos del “Do It Yourself” al “Do It With Others”. *Este autoaprendizaje colectivo* implica un hacer autodidacta involucrando a los demás, a una comunidad, a un curso, a todos.
"collective self-learning :))

**Comment by patcon** [comment]
One more concept on my mind lately, that might be relevant:
- Making **altruistic behaviour legible** in complex environments

This is maybe springboarding off my fave point in @dcwalk's list:

> - lets do it with others (DIWO) (as opposed to DIY?)

tl;dr - walked by a man pulling a dolly up stairs yesterday. saw he was having trouble. asked whether he could use help. he consented. i grabbed the bottom and nudged it. he fell over. he got up. we got better at it. we were both smiling and appreciative by the end.

the above is simple and intuitive human-scale exchange involved objects, in classic physical space between two people. but what if one of those people is an organization? where are the edges and the corners to see and recognize as requiring assistance? how does the offereing individual know to offer help? How does the organizing group understand and consent to that help?

and nevermind if both parties are groups. that's hella hard to intuit

altruistic actions gets complicated in these situations, but maybe we could riff on how it could be made simpler?

**Comment by dcwalk** [comment]
Okay, it seems like the clear winner is DIWO! I'm gonna do some writing thie early aft in this doc: https://docs.google.com/document/d/14-N79UdKsayX995NPaNj4a2snVqs3aP2rFS2MfKoHVI/edit# (sorry academic-mode). Our deadline on a final version is end of next week (see #9), once we have a solid theme section, which I anticipate before the end of the weekend, we can close this.

**Comment by dcwalk** [comment]
We went ahead with the conf theme 🎉 , final polishing of CFP will he tracked in #9 and getting up on the website in #25
